spec/forms/outcome_date_form_spec.rb

```diff
@@ -3,6 +3,8 @@
 require "rails_helper"
 
 describe OutcomeDateForm, type: :model do
+  include ActiveJob::TestHelper
```

Comment:
I had problems with `have_enqueued_job` always failing on CI until I added this. I'm not 100% sure why that is because the `have_enqueued_job` matcher ran (it wasn't a missing method), it just didn't pass.